controllers/object_controls.go

```diff
@@ -147,6 +148,8 @@ const (
 	NvidiaCtrRuntimeCDIPrefixesEnvName = "NVIDIA_CONTAINER_RUNTIME_MODES_CDI_ANNOTATION_PREFIXES"
 	// CDIEnabledEnvName is the name of the envvar used to enable CDI in the operands
 	CDIEnabledEnvName = "CDI_ENABLED"
+	// NvidiaCTKHookPathEnvName is the name of the envvar specifying the path to the 'nvidia-ctk' binary
```

Comment:
Comment is incorrect.

Reply:
Note we shouldn't need the `nvidia-ctk` path in addition to the `nvidia-cdi-hook` path. They can be used interchangeably.

Reply:
Updated the comment. Will remove this once https://github.com/NVIDIA/k8s-dra-driver/issues/210 is complete.